tests/playwright/helpers/wordpress.js

```diff
@@ -0,0 +1,206 @@
+const { execSync } = require('child_process');
+
+/**
+ * Core WordPress Test Helpers
+ * 
+ * Utilities for core WordPress functionality: authentication, admin navigation,
+ * permalinks, and basic WordPress operations.
+ */
+
+const { Admin, PageUtils } = require('@wordpress/e2e-test-utils-playwright');
+
+/**
+ * Wait for WordPress admin to be ready
+ * 
+ * @param {import('@playwright/test').Page} page - Playwright page object
+ * @returns {Promise<void>}
+ */
+async function waitForWordPressAdmin(page) {
+  // Wait for WordPress admin bar or body class
+  await page.waitForSelector('body.wp-admin', { timeout: 10000 });
+  
+  // Wait for admin menu to be visible
+  await page.waitForSelector('#adminmenu', { timeout: 5000 });
+}
+
+/**
+ * Check if user is logged into WordPress
+ * 
+ * @param {import('@playwright/test').Page} page - Playwright page object
+ * @returns {Promise<boolean>} True if logged in
+ */
+async function isLoggedIn(page) {
+  try {
+    await page.goto('/wp-admin/');
+    await page.waitForSelector('body.wp-admin', { timeout: 3000 });
+    return true;
+  } catch {
+    return false;
+  }
+}
+
+/**
+ * Get WordPress admin menu items
+ * 
+ * @param {import('@playwright/test').Page} page - Playwright page object
+ * @returns {Promise<Array>} Array of menu item objects
+ */
+async function getAdminMenuItems(page) {
+  const menuItems = await page.locator('#adminmenu li').all();
+  const items = [];
+  
+  for (const item of menuItems) {
+    const text = await item.textContent();
+    const href = await item.locator('a').getAttribute('href');
+    items.push({ text: text?.trim(), href });
+  }
+  
+  return items;
+}
+
+/**
+ * Navigate to plugin page
+ * 
+ * @param {import('@playwright/test').Page} page - Playwright page object
+ * @param {string} pluginSlug - Plugin slug (e.g., 'bluehost')
+ * @param {string} subPage - Sub-page hash (e.g., '#/settings')
+ * @returns {void}
+ */
+async function navigateToPluginPage(page, pluginSlug, subPage = '') {
+  const pageUtils = new PageUtils({ page });
+  const admin = new Admin({ page, pageUtils });
+  
+  await admin.visitAdminPage(`admin.php?page=${pluginSlug}${subPage}`);
+}
+
+/**
+ * Check if plugin is active
+ * 
+ * @param {import('@playwright/test').Page} page - Playwright page object
+ * @param {string} pluginSlug - Plugin slug
+ * @returns {boolean} True if plugin is active
+ */
+async function isPluginActive(page, pluginSlug) {
+  const pageUtils = new PageUtils({ page });
+  const admin = new Admin({ page, pageUtils });
+  
+  await admin.visitAdminPage('plugins.php');
+  
+  const pluginRow = page.locator(`tr[data-plugin*="${pluginSlug}"]`);
+  const deactivateLink = pluginRow.locator('a[href*="deactivate"]');
+  
+  return await deactivateLink.isVisible();
+}
```

Comment:
How is this different from the isPluginActive() in tests/playwright/helpers/newfold.js?

Reply:
Duplicate, thanks for spotting it. Removed.